src/components/Trip/TripFullPage/TripFullPage.jsx

```diff
@@ -66,6 +69,50 @@ const TripFullPage = () => {
 		setParticipantsData(participantsFilteredArray);
 	};
 
+	const getTripDuration = () => {
+		let tripDurationText = '';
+		const startDay = trip.startDate.toDate().getDate();
+		const startMonth = trip.startDate.toDate().getMonth();
+		const startYear = trip.endDate.toDate().getFullYear();
+
+		const endDay = trip.endDate.toDate().getDate();
+		const endMonth = trip.endDate.toDate().getMonth();
+		const endYear = trip.endDate.toDate().getFullYear();
+
+		if (startYear === endYear && startMonth === endMonth) {
+			tripDurationText = startDay + '-' + endDay + '.' + endMonth + '.' + endYear;
+		} else if (startYear === endYear) {
+			tripDurationText =
+				startDay + '.' + startMonth + '-' + endDay + '.' + endMonth + '.' + endYear;
+		} else {
+			tripDurationText =
+				startDay +
+				'.' +
+				startMonth +
+				'.' +
+				startYear +
+				'-' +
+				endDay +
+				'.' +
+				endMonth +
+				'.' +
+				endYear;
+		}
+		setTripDuration(tripDurationText);
```

Comment:
Jeśli dobrze widzę, to zmienia miesiąc na o jeden wcześniej, ale nie widzę dlaczego

Reply:
Poprawiłam. Dokumentacja mówi, że getMonth() po prostu liczy miesiące od zera.